packages/@react-spectrum/tooltip/stories/TooltipTrigger.stories.tsx

```diff
@@ -251,6 +256,24 @@ function renderMultipleTriggers(props = {}) {
   );
 }
 
+function ActionGroupTrigger() {
+  let [key, setKey] = useState('');
+  return (
+    <Flex direction="column">
+      <div>Click the trigger to log the trigger's key:{key}</div>
+      <ActionGroup
+        selectionMode="single"
+        disallowEmptySelection
+        onSelectionChange={() => setKey('Key')}>
+        <TooltipTrigger delay={0}>
+          <Item key={key}>Trigger Tooltip</Item>
+          <Tooltip>Tooltip is inside an ActionGroup</Tooltip>
+        </TooltipTrigger>
+      </ActionGroup>
+    </Flex>    
+  );
+}
+
```

Comment:
Hm this isn't quite right, you'll want to have a static key set on the item and then provide `setKey` as is to `onSelectionChange` so that `key` is properly updated to the latest ActionGroup `Item` you press. My previous example was probably a bit confusing since I used the term "key" everywhere and prob didn't have the right types, that's my bad. I've refined it so that you can use storybook's action addon instead

```suggestion
function ActionGroupTrigger() {
  let onSelectionChange = action('onSelectionChange');

  return (
    <ActionGroup
      selectionMode="single"
      disallowEmptySelection
      onSelectionChange={s => onSelectionChange([...s])}>
      <TooltipTrigger delay={0}>
        <Item key="item 1">Trigger Tooltip</Item>
        <Tooltip>Tooltip is inside an ActionGroup</Tooltip>
      </TooltipTrigger>
    </ActionGroup>
  );
}
```

Reply:
I Implemented the Suggestion